fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -436,6 +436,19 @@ public void updateProperties(final IdentifierConverter<Resource, FedoraResource>
 
         final Collection<IllegalArgumentException> errors = checkInvalidPredicates(request);
 
+        request.getPrefixMapping().getNsPrefixMap().forEach(
+                       (k,v) -> { try {
+                                   LOGGER.info("451: prefix mapping is key -> value: " + k + "->" + v + "\n");
```

Comment:
Comments on your logging:
* When using slf4j (which you are), you should take advantage of templating. For example, the above line should be:
`LOGGER.info("451: prefix mapping is key -> value: {} -> {}", k, v);`
* This should be `debug` not `info`